components/Common/LanguageSelector/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { useMemo, useState } from 'react';
+import { MdOutlineTranslate } from 'react-icons/md';
+import { useLocale } from '../../../hooks/useLocale';
+import Dropdown from '../Dropdown';
+import styles from './index.module.scss';
+
+const dropdownStyle = {
+  position: 'absolute',
+  top: '60%',
+  right: '0',
+  margin: 0,
+};
+
+const LanguageSelector = () => {
+  const [showDropdown, setShowDropdown] = useState(false);
+
+  const { availableLocales, currentLocale } = useLocale();
+
+  const dropdownItems = useMemo(
+    () =>
+      availableLocales.map(locale => ({
+        title: locale.localName,
+        label: locale.name,
+        onClick: () => {},
+        active: currentLocale.code === locale.code,
+      })),
+    [availableLocales, currentLocale]
+  );
+
+  return (
+    <div className={styles.container}>
+      <button
+        type="button"
+        className={styles.languageSwitch}
+        onClick={() => {
```

Comment:
Nitpick: `onClick{() => setShowDropdown(!showDropdown);`

Reply:
Prettier not allowing in single line

Reply:
How so? What happens when you do `onClick={() => setShowDropdown(!showDropdown)}`?
(Also it doesn't need to be in one line, the point here is to remove the brackets.

Reply:
This is to toggle the dropdown on and off. we are inverting the boolean variable and updating the state so that dropdown will be visible when hidden and vice versa

Reply:
I know what it does. I'm just asking you to remove the `{}` brackets surrounding the `onClick`